.github/workflows/nightly.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           - 'latest'
         refs:
           - 'master'
-          - '0.38'
+          - '0.39'
```

Comment:
This needs to be adjusted on three files:

* `nightly-helm.yml`
* `nightly-kops.yml`
* `nightly.yml`

Reply:
thanks, updated.